setup.py

```diff
@@ -55,10 +55,7 @@ def get_version():
     "Operating System :: Microsoft :: Windows",
```

Comment:
From 4 years ago, the tests only run on `ubuntu-latest`. We could either test on Mac and Windows as well, or stop listing support here, but I don't think it's valid to say that they're supported when they're not tested.

Reply:
I am trying a `matrix` strategy to run on `ubuntu-latest`, `macos-latest`, and `windows-latest` now.